imagebuildah/build.go

```diff
@@ -1613,6 +1613,16 @@ func (s *StageExecutor) EnsureContainerPath(path string) error {
 	if err != nil && os.IsNotExist(err) {
 		err = os.MkdirAll(filepath.Join(s.mountPoint, path), 0755)
 	}
+	// Let's see if we're dealing with a symlink
+	if err != nil {
+		var sympath string
+		sympath, err = filepath.EvalSymlinks(path)
```

Comment:
Should this be on path or s.mountPoint+path?
Should you do this before you do the MkDirALl above, so we only have one call to MkdirALL?

Reply:
Oh shoot, I think your right about joining them.  I've reworked the logic a bit, only one mkdireall now.  I'll repush later.